### ISSUE TITLE
Suggetion : Use Bootstrap Popover Plugin to display messages

### ISSUE THREAD

**Issue by teranchristian** [Chiliec/yii2-vote#25]
Currently, the messages are display using DIVs. This might break the layout of the site.

**Before vote**
![screenshot from 2016-05-09 19-25-00](https://cloud.githubusercontent.com/assets/5291283/15108885/cdf9f556-161b-11e6-90b2-6cc139b71ed4.png)

**After vote**
![screenshot from 2016-05-09 19-25-11](https://cloud.githubusercontent.com/assets/5291283/15108898/e3791236-161b-11e6-8695-5acc3093e006.png)

displaying the messages using the popover plugin can fix the layout issue
**Example**
![screenshot from 2016-05-09 19-28-14](https://cloud.githubusercontent.com/assets/5291283/15108956/3e32e698-161c-11e6-85d9-76ad63110201.png)

**Comment by chiliec** [comment]
Good suggestion, I like it.

**Comment by teranchristian** [comment]
I'm already working on it. Happy to share my code

**Comment by loveorigami** [comment]
I have new way with notification.
-  In vote action https://github.com/Chiliec/yii2-vote/blob/master/actions/VoteAction.php#L23 all messages I return as flash-message

``` php
\Yii::$app->session->setFlash('error', Yii::t('vote', 'modelId has not been sent'));
```

example - https://github.com/loveorigami/lo-module-vote/blob/master/actions/VoteAction.php
- Next I use yii2-notification wrapper
  https://github.com/loveorigami/yii2-notification-wrapper

where I can use different Layers for display flash-notification.
https://github.com/loveorigami/yii2-notification-wrapper#supported-layers

**Comment by teranchristian** [comment]
this check this https://github.com/teranchristian/yii2-vote, still a couple  of things to improve

**Comment by jsobers1331** [comment]
@teranchristian which line in your code is responsible for the bootstrap popup in your screen shot above?

**Comment by teranchristian** [comment]
Hi @jsobers1331, did you try the code ?
about the code  look at yii2-vote/widget/vote.php

**Comment by jsobers1331** [comment]
Thank you, I found it in your popover branch.

**Comment by chiliec** [comment]
@teranchristian why you don't create pull request?

**Comment by teranchristian** [comment]
hey @Chiliec, sure. I am going to clean the code tomorrow, then i will pull a request. Cheers, C

**Comment by chiliec** [comment]
Great work, I merged, thanks! But only I have this problem?
![strange](https://cloud.githubusercontent.com/assets/982358/16926773/aac75a42-4d43-11e6-82a3-8f73cffe6596.gif)
P.S.: Chrome 51.0.2704.106 on Mac OS El Captain 10.11.5

**Comment by jsobers1331** [comment]
@teranchristian could you take a look above? would love this in my project.

**Comment by teranchristian** [comment]
Hey @jsobers1331,
sorry for my late reply. 
I just fixed this issue and pushed my changes. 
Do you mind testing it ?
cheers,
C

**Comment by chiliec** [comment]
I've merged. Thank you, @teranchristian!

**Comment by chiliec** [comment]
Included in [release 4.0.0](https://github.com/Chiliec/yii2-vote/releases/tag/4.0.0)
